content/news/XXXX-XX-XX-mixxx-2_5-released .md

```diff
@@ -0,0 +1,541 @@
+title: Mixxx 2.5 Release Announcement
+authors: Evelynne Veys
+tags: 2.5, release announcement
+comments: yes
+status: draft
+
+#### All I want for DJ-ing is ...
+
+Home alone? Scrooged? The sound of music? Die hard?
+Change plans: this year you will be having something better to do than watching old Christmas movies.
+Mixxx is now your secret "Santa comin' to town", our elves had "good hearts with the burning desire to work hard",
+to get you "Rockin' around the Christmas tree at the party hop" with "something mighty sweet to see'.
+"For the present, look high, not low": Mixxx's "presents 'neath the tree"
+
+#### So this is the last goodbye ...
+
+Important: OS-Support.
+macOS versions earlier than 11 can have a "Last Christmas".
+Windows versions earlier than version 10 build 1809 can be "Driving home for Christmas".
+For Ubunto versions earlier than 22.04 "Christmas Will Break Your Heart".

```

Comment:
Metaphors are fine for entertainment value, but we should not rely on them exclusively for conveying information. 
```suggestion
macOS versions earlier than 11 can have a "Last Christmas".
Windows versions earlier than version 10 build 1809 can be "Driving home for Christmas".
For Ubuntu versions earlier than 22.04 "Christmas Will Break Your Heart".
Unfortunately support for all this version has been dropped for Mixxx 2.5 due to <insert explanation here>. 
```